meta-freescale-mbl/recipes-bsp/u-boot/u-boot-fslc/imx7d-pico-u-boot-dtb.cfgout

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2018-2019 Arm Limited and Contributors. All rights reserved.
```

Comment:
where did this file come from?

if this file is going to be called imx7d-pico-uboot-dtb.cfg, then the warp7 file should be called imx7s-warp-u-boot-dtb.cfg? I suggest you call the files ${MACHINE}-u-boot-dtb.cfg and make one do_deploy_depends() which is uses the suggested naming convention.

Reply:
This file is written by me, with values abstracted from u-boot SPL code and modified with referencing warp7 dcd file.

I can change the name. But dcd for warp7 is from ${B}, while dcd for pico is from ${WORKDIR}. How can we align the path, creating DCD_PATH_imx7s-warp-mbl DCD_PATH_imx7d-pico-mbl is acceptable?

Reply:
>>This file is written by me, with values abstracted from u-boot SPL code 

The filel is therefore derived from u-boot spl code and that needs to be acknowledged in the licensing. See here for details:

https://confluence.arm.com/pages/viewpage.action?pageId=266055215

and this within the page: 

```
Based on: (OSS-FILE)
In open-source project: (OSS-URI)
 
Original file: (OSS-COPYRIGHT)
Modifications: Copyright (c) 2018 Arm Limited and Contributors. All rights reserved.
 
SPDX-License-Identifier: (OSS-LICENSE-SPDX)
```

>> reating DCD_PATH_imx7s-warp-mbl DCD_PATH_imx7d-pico-mbl is acceptable?

yes

Reply:
Done